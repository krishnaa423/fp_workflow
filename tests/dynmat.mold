[Molden Format]
[FREQ]
    0.00
    0.00
    0.00
  584.40
  584.40
  584.40
[FR-COORD]
   Si          0.00000        0.00000        0.00000
   Si          2.56530        2.56530        2.56530
[FR-NORM-COORD]
 vibration     1
   0.13370  -0.17136   0.67287
   0.13370  -0.17136   0.67287
 vibration     2
  -0.41975   0.52590   0.21734
  -0.41975   0.52590   0.21734
 vibration     3
  -0.55311  -0.44053  -0.00229
  -0.55311  -0.44053  -0.00229
 vibration     4
   0.00000   0.00000  -0.70711
   0.00000  -0.00000   0.70711
 vibration     5
  -0.67082  -0.22361  -0.00000
   0.67082   0.22361  -0.00000
 vibration     6
   0.22361  -0.67082   0.00000
  -0.22361   0.67082   0.00000
