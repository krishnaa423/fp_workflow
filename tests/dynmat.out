     diagonalizing the dynamical matrix ...

 q =       0.0000      0.0000      0.0000
 **************************************************************************
     freq (    1) =      -0.000000 [THz] =      -0.000002 [cm-1]
 (  0.133700  -0.000000    -0.171364   0.000000     0.672874   0.000000   )
 (  0.133700  -0.000000    -0.171364   0.000000     0.672874   0.000000   )
     freq (    2) =       0.000000 [THz] =       0.000005 [cm-1]
 ( -0.419753   0.000000     0.525900   0.000000     0.217338  -0.000000   )
 ( -0.419753   0.000000     0.525900   0.000000     0.217338  -0.000000   )
     freq (    3) =       0.000000 [THz] =       0.000008 [cm-1]
 ( -0.553111   0.000000    -0.440526   0.000000    -0.002288   0.000000   )
 ( -0.553111   0.000000    -0.440526   0.000000    -0.002288   0.000000   )
     freq (    4) =      17.519899 [THz] =     584.400932 [cm-1]
 (  0.000000   0.000000     0.000000   0.000000    -0.707107   0.000000   )
 (  0.000000   0.000000    -0.000000   0.000000     0.707107   0.000000   )
     freq (    5) =      17.519899 [THz] =     584.400932 [cm-1]
 ( -0.670820   0.000000    -0.223607   0.000000    -0.000000   0.000000   )
 (  0.670820   0.000000     0.223607   0.000000    -0.000000   0.000000   )
     freq (    6) =      17.519899 [THz] =     584.400932 [cm-1]
 (  0.223607   0.000000    -0.670820   0.000000     0.000000   0.000000   )
 ( -0.223607   0.000000     0.670820   0.000000     0.000000   0.000000   )
 **************************************************************************
